multipoint/multiPointSparse.py

```diff
@@ -163,7 +163,7 @@ def addProcessorSet(self, setName, nMembers, memberSizes):
                 memberSizes = numpy.ones(nMembers) * memberSizes[0]
             else:
                 if len(memberSizes) != nMembers:
-                    raise MPError("The suppliled memberSizes list is not the correct length.")
+                    raise MPError("The suppllied memberSizes list is not the correct length.")
 
```

Comment:
same here

Reply:
This is also fixed in a subsequent commit I believe, so the only thing I'll update is the year.

Reply:
Oh no you're right, sorry. Will fix this